Chapters/3-7-Chapter21.md

```diff
@@ -0,0 +1,501 @@
+
+He turned toward everyone in the room as they sat looking to him and Melna. "We are about to make history as the first people to make contact with life outside of our own planet. You all play a vital role to the survival of our species. In our mission, a mistake will cost you your own life, your families lives. Everyone you know will cease to exist."
+
+He held a moment of dramatic silence
+
+"For Ara!" he growled.
+
+And the room bellowed in unison, "For Ara!"
+
+He gestured with his hand in a twirling motion and they all returned to their bulbs and paper.
+
+"Operator, public," he ordered.
+
+Thi flipped a switch and the room filled with static and the sound of my breath in the capsule. Fajro lifted the microphone to his mouth again and glanced back at Melna to ensure she saw that he was in control.
+
+"Liffa, you're about to pass to the other side of the obscura. Describe to us what you see."
+
+"Y-Yes sir," I said. "I've been watching the obscura closely now. It... looks a bunch of circles. They're all growing outward from their centers."
+
+"And how large are these circles?"
+
+"No larger than a train car. But there are so many. More than I can see. More than I can count."
+
+"Can you see the beings?," he asked pointedly.
+
+"I... don't know."
+
+"You have to give us more than that."
+
+"I can't see very far," I told the truth. "My vision... It's blurry."
+
+He was doubtful, but continued.
+
+"Just tell me what you can see then."
+
+"I'm just a few thousand paces out now; velocity relative to the obscura is low... I think I can see some detail in the shapes."
+
+"What do you see?" He spurted with an eager breath.
+
+"These circles, they have barely any thickness at all."
+
+"There must be something building them."
+
+"They appear to be... unraveling. Like some sort of fabric."
+
+Every eye in the room looked up at once, silently processing what such a thing could mean. For themselves, for their families. They were all convinced that this enemy lived like them, thought like them: hostile and intelligent, surviving within the confines of whatever planet where they came from. But no one knew what to make of this.
+
+"Liffa, have you tried the signal?"
+
+"I tried it a bit further out. Let me try again."
+
+Fajro took his finger off of the microphone lever and yelled across the room, "Kavva, how close is Liffa to the obscura?"
+
+"About 800 paces, sir," he yelle in response.
+
+He paused a moment then with a hurried voice he said, "Liffa, is it done? What do you see?"
+
+"The fabric... has torn. Right where I'm headed."
+
+"Torn?"
+
+"I can see Ara now. Oh it's so bright! And the shapes, they don't end... they're -o..."
+
+"Liffa? Liffa, are you there?"
+
+The line went turbulent with static and shreds of Liffas voice.
+
+Melna wanted to cry out hae name, to embrace hae and tell hae it was okay; that thi didn't have to face the end alone. But the stinging pain around her neck took her words, like Ramne had taken her agency.
```

Comment:
> like Ramne had taken her agency.

edit mishap? according to your notes you had taken the scene you are referring to out.